package.json

```diff
@@ -32,6 +32,7 @@
     "test-ci": "cross-env NODE_ENV=test npm run lint && npm run flow && npm run test"
   },
   "dependencies": {
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
```

Comment:
should this be in `devDependencies` since it should only be needed for dev/build? that's where the other babel stuff is pulled in

Reply:
Good point, I'll change it.